### ISSUE TITLE
Can't access metric data via REST interface

### ISSUE THREAD

**Issue by pmembrey** [boundary/folsom_cowboy#2]
Hi,

I'm guessing this could be an incompatibility between folsom and folsom_cowboy, but folsom_cowboy doesn't have any tags, so I'm not sure which version I am supposed to use :)

I am able to create a meter metric called msg_per_sec:

``` erlang
folsom_metrics:new_meter(msg_per_sec)
```

Then I update it every time I receive a new message:

``` erlang
folsom_metrics:notify({msg_per_sec, 1}),
```

I can see the stats are being collected okay because get_metric_value works fine:

``` erlang
5> folsom_metrics:get_metric_value(msg_per_sec).  

[{count,2257},
 {one,9.793870473767395},
 {five,7.6517299033567285},
 {fifteen,6.22315210736992},
 {mean,9.818651166080544e-6},
 {acceleration,[{one_to_five,0.007140468568035556},
                {five_to_fifteen,0.0023809629933113476},
                {one_to_fifteen,0.003967464851552751}]}]

```

If I try to look at the available metrics from a browser with "http://localhost:5565/_metrics" I get:

```
["msg_per_sec"]
```

But if I try "http://localhost:5565/_metrics/msg_per_sec" I get a 500 error with this in the Erlang console:

``` erlang

14:05:24.844 [error] ** Handler folsom_cowboy_metrics_handler terminating in handle/2
   for the reason error:undef
** Options were []
** Handler state was undefined
** Request was [{socket,#Port<0.4793>},{transport,cowboy_tcp_transport},{connection,keepalive},{pid,<0.371.0>},{method,'GET'},{version,{1,1}},{peer,undefined},{host,[<<"localhost">>]},{host_info,undefined},{raw_host,<<"localhost">>},{port,5565},{path,[<<"_metrics">>,<<"msg_per_sec">>]},{path_info,undefined},{raw_path,<<"/_metrics/msg_per_sec">>},{qs_vals,undefined},{raw_qs,<<>>},{bindings,[]},{headers,[{'Accept-Charset',<<"ISO-8859-1,utf-8;q=0.7,*;q=0.3">>},{'Accept-Language',<<"en-US,en;q=0.8">>},{'Accept-Encoding',<<"gzip,deflate,sdch">>},{'Accept',<<"text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8">>},{'User-Agent',<<"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_7_4) AppleWebKit/536.11 (KHTML, like Gecko) Chrome/20.0.1132.57 Safari/536.11">>},{'Connection',<<"keep-alive">>},{'Host',<<"localhost:5565">>}]},{p_headers,[{'Connection',[<<"keep-alive">>]}]},{cookies,undefined},{meta,[]},{body_state,waiting},{buffer,<<>>},{resp_state,waiting},{resp_headers,[]},{resp_body,<<>>},{onresponse,undefined},{urldecode,{#Fun<cowboy_http.urldecode.2>,crash}}]
** Stacktrace: [{mochinum,digits,[9.889871914675805],[]},{mochijson2,'-json_encode_proplist/2-fun-0-',3,[{file,"src/mochijson2.erl"},{line,167}]},{lists,foldl,3,[{file,"lists.erl"},{line,1197}]},{mochijson2,json_encode_proplist,2,[{file,"src/mochijson2.erl"},{line,170}]},{mochijson2,'-json_encode_proplist/2-fun-0-',3,[{file,"src/mochijson2.erl"},{line,167}]},{lists,foldl,3,[{file,"lists.erl"},{line,1197}]},{mochijson2,json_encode_proplist,2,[{file,"src/mochijson2.erl"},{line,170}]},{folsom_cowboy_metrics_handler,get_request,1,[{file,"src/folsom_cowboy_metrics_handler.erl"},{line,45}]}]
```

Basically I'm not sure whether it's something I'm doing wrong, a version compatibility issue or something else :)
